src/gt4py/cartesian/backend/dace_backend.py

```diff
@@ -561,13 +561,21 @@ def apply(cls, stencil_ir: gtir.Stencil, builder: "StencilBuilder", sdfg: dace.S
         else:
             omp_threads = ""
             omp_header = ""
+
+        # Backward compatible state struct name change in dave >=0.15.x
+        try:
```

Comment:
Would it be ok from your side if we drop support for dace < 0.15.0? If not, we can bump the minimum required version to 0.15.x

Reply:
Not quite yet, unfortunately. I will log it as a task on our end for near-term when we flush out the first GEOS GPU version, then revisit this code and kill it.

Reply:
Logged NASA side